packages/react-test-renderer/src/ReactShallowRenderer.js

```diff
@@ -674,8 +682,13 @@ class ReactShallowRenderer {
     }
 
     this._rendered = this._instance.render();
-    // Intentionally do not call componentDidMount()
+    // Force user to opt in to calling componentDidMount()
     // because DOM refs are not available.
+    if(this._useLifecycleMethods) {
```

Comment:
spacing

Reply:
I'm still trying to get feedback on the implementation plan. I'll clean up spacing and things like that at the end